modules/graphics/demo-apps.nix

```diff
@@ -0,0 +1,72 @@
+# Copyright 2022-2023 TII (SSRC) and the Ghaf contributors
+# SPDX-License-Identifier: Apache-2.0
+{
+  pkgs,
+  lib,
+  config,
+  ...
+}: let
+  cfg = config.ghaf.graphics.demo-apps;
+  weston = config.ghaf.graphics.weston;
+
+  /* Scaled down firefox icon */
+  firefox-icon = pkgs.runCommand "firefox-icon-24x24" { } ''
+    mkdir -p $out/share/icons/hicolor/24x24/apps
+    ${pkgs.buildPackages.imagemagick}/bin/convert \
+      ${pkgs.firefox}/share/icons/hicolor/128x128/apps/firefox.png \
+      -resize 24x24 \
+      $out/share/icons/hicolor/24x24/apps/firefox.png
+  '';
+
+  /*
+  Generate launchers to be used in weston.ini
+
+  Type: mkProgramOption ::  string -> bool -> option
+
+  */
+  mkProgramOption = name: default:
+    with lib;
+      mkOption {
+        inherit default;
+        type = types.bool;
+        description = "Include package ${name} to menu and system environment";
+      };
+in {
+  options.ghaf.graphics.demo-apps = with lib; {
+    chromium = mkProgramOption "Chromium browser" false;
+    firefox = mkProgramOption "Firefox browser" weston.enableDemoApplications;
+    gala-app = mkProgramOption "Gala App" false;
```

Comment:
Why is gala false by default?

Reply:
gala require an account. And @vilvo suggest me to off it by default. Could be overriden by downstream projects

Reply:
I guess we can disable it now and then enable it again when someone asks for it